chroma-manager/chroma_core/migrations/0038_auto__add_primary_umtm_steps.py

```diff
@@ -0,0 +1,1503 @@
+# -*- coding: utf-8 -*-
+import cPickle as pickle
```

Comment:
Is this import needed?

Reply:
it's a better performing implementation and I have been in the habit of using it in preference to the original implementation

Reply:
Sure, I just meant I don't see `pickle` being used directly anywhere.